packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -742,14 +742,15 @@ export class TeamsBotImpl {
       Logger.info(Messages.ProvisioningBotRegistration);
       botAuthCreds = await AADRegistration.registerAADAppAndGetSecretByAppStudio(
         appStudioToken!,
-        aadDisplayName
+        aadDisplayName,
+        this.config.localDebug.localObjectId
```

Comment:
The overall logic is changed significantly. I recommend to refactor the code to describe the new logic instead of patching on the legacy version.

Reply:
For this PR, I think no need.